external/storm-jdbc/README.md

```diff
@@ -3,14 +3,16 @@ Storm/Trident integration for JDBC. This package includes the core bolts and tri
 to either insert storm tuples in a database table or to execute select queries against a database and enrich tuples 
 in a storm topology.
 
+**Note**: Throughout the examples below, we make use of com.google.common.collect.Lists and com.google.common.collect.Maps.
+
 ## Inserting into a database.
 The bolt and trident state included in this package for inserting data into a database tables are tied to a single table.
 
 ### ConnectionProvider
-An interface that should be implemented by different connection pooling mechanism `org.apache.storm.jdbc.common.ConnectionProvider`
+An interface that should be implemented by different connection pooling mechanism `org.apache.storm.jdbc.common.ConnectionPrvoider`
```

Comment:
I cloned from master, not sure where this change came from. Corrected in the next commit.